docs/migration/01_migration_guide_0.5.ipynb

```diff
@@ -0,0 +1,884 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "tough-timeline",
+   "metadata": {},
+   "source": [
+    "![new_qk.png](attachment:new_qk.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fatal-forty",
+   "metadata": {},
+   "source": [
+    "The new kernels expose the same interface and the same parameters except the `quantum_instance` parameter. This parameter does not have a direct replacement and instead the `fidelity` parameter must be used.\n",
```

Comment:
Since quantum_instance and fidelty seem like such different things should we note that the backend handling/selection, which was previously done using the quantum instance, is now taken care of via the Estimator primitive given to the fidelity. So an execution environment is still given just a bit more indirectly with the fidelity - it now handle the circuit execution for that.

Reply:
Added your text about handling/selection.